JSDemos/Demos/Toolbar/Multiline/jQuery/index.js

```diff
@@ -0,0 +1,370 @@
+    options: {
+      items: [
+        {
+          icon: 'orderedlist',
+          alignment: 'orderedlist',
+          hint: 'Ordered',
+        },
+        {
+          icon: 'bulletlist',
+          alignment: 'bulletlist',
+          hint: 'Bullet',
+        },
+      ],
+      keyExpr: 'alignment',
+      stylingMode: 'outlined',
+      onItemClick(e) {
+        DevExpress.ui.notify(`The "${e.itemData.hint}" button was clicked`);
+      },
+    },
+  },
+  {
+    cssClass: 'menu-separator-container',
+    locateInMenu: 'auto',
+    location: 'before',
+    template(itemData, itemIndex, element) {
+      $(element).addClass('separator');
+    },
+    menuItemTemplate(itemData, itemIndex, element) {
+      $(element).addClass('menu-separator');
+    },
+  },
+  {
+    locateInMenu: 'auto',
+    location: 'before',
+    widget: 'dxDateBox',
+    options: {
+      width: 200,
+      type: 'date',
+      value: new Date(2022, 9, 7),
+    },
+  },
+  {
+    cssClass: 'menu-separator-container',
+    locateInMenu: 'auto',
+    location: 'before',
+    template(itemData, itemIndex, element) {
+      $(element).addClass('separator');
+    },
+    menuItemTemplate(itemData, itemIndex, element) {
+      $(element).addClass('menu-separator');
+    },
+  },
+  {
+    locateInMenu: 'auto',
+    location: 'before',
+    widget: 'dxCheckBox',
+    options: {
+      value: false,
+      text: 'Checkbox text',
+      onOptionChanged() {
+        DevExpress.ui.notify('Checkbox option has been changed!');
+      },
+    },
+  },
+  {
+    location: 'after',
+    widget: 'dxButton',
+    showText: 'inMenu',
+    options: {
+      icon: 'attach',
+      text: 'Attach',
+      onClick() {
+        DevExpress.ui.notify('Attach button has been clicked!');
+      },
+    },
+  },
+  {
+    locateInMenu: 'auto',
+    location: 'after',
+    widget: 'dxButton',
+    showText: 'inMenu',
+    options: {
+      icon: 'add',
+      text: 'Add',
+      onClick() {
+        DevExpress.ui.notify('Add button has been clicked!');
+      },
+    },
+  },
+  {
+    locateInMenu: 'auto',
+    location: 'after',
+    widget: 'dxButton',
+    showText: 'inMenu',
+    options: {
+      icon: 'trash',
+      text: 'Remove',
+      onClick() {
+        DevExpress.ui.notify('Remove button has been clicked!');
+      },
+    },
+  },
+  {
+    locateInMenu: 'always',
+    widget: 'dxButton',
+    showText: 'inMenu',
+    options: {
+      icon: 'help',
+      text: 'About',
+      onClick() {
+        DevExpress.ui.notify('About button has been clicked!');
+      },
+    },
+  },
+];
+
+$(() => {
+  $('#resizable-container').dxResizable({
+    minWidth: 500,
+    minHeight: 150,
+    maxHeight: 370,
+    handles: 'right',
+  });
+
+  const toolbar = $('#toolbar').dxToolbar({
+    dataSource: toolbarDataDefault,
+    multiline: true,
+  }).dxToolbar('instance');
+
+  const toolbarLineModes = ['Multiline mode', 'Singleline mode'];
+
+  $('#toolbar-modes').dxRadioGroup({
```

Comment:
Why we don't use the approach like this

```
 $('#radio-group-simple').dxRadioGroup({
    items: [{
      text: "multiline",
      value: true
    }, {
      text: "singleline",
      value: false
    }],
    value: true,
    valueExpr: 'value'
  });
```

In this case in all approaches we avoid using additional state variable and may be used the two way binding in angular

Reply:
@ksercs what do you think?

Reply:
See: https://github.com/DevExpress/devextreme-demos/pull/2004/commits/8a83a8ab0b90e53c18eb3b0732e0f89877c34e6a